app/client/cli/peer/connections.go

```diff
@@ -0,0 +1,80 @@
+package peer
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"google.golang.org/protobuf/types/known/anypb"
+
+	"github.com/pokt-network/pocket/app/client/cli/helpers"
+	"github.com/pokt-network/pocket/p2p/debug"
+	"github.com/pokt-network/pocket/shared/messaging"
+)
+
+var (
+	connectionsCmd = &cobra.Command{
+		Use:   "connections",
+		Short: "Print open peer connections",
+		RunE:  connectionsRunE,
+	}
+)
+
+func init() {
+	PeerCmd.AddCommand(connectionsCmd)
+}
+
+func connectionsRunE(cmd *cobra.Command, _ []string) error {
+	var routerType debug.RouterType
+
+	bus, err := helpers.GetBusFromCmd(cmd)
+	if err != nil {
+		return err
+	}
+
+	switch {
+	case stakedFlag:
```

Comment:
OPTIONAL: The way I would've done this is by having a smaller private helper function like below that returns the router typ:

```go
func foo() routerType {
	switch
	case stakedFlag && !unstakedFlag && !allFlag {
		return debug.StakeRouterType
	}
	case unstakedFlag && !stakedFlag && !allFlag {
		return debug.UnstakedRouterType
	}
	case stakedFlag || unstakedFlag {
		return ErrRouterType
	}
	// even if `allFlag` is false, we still want to print all connections
	default:
		return debug.AllRouterTypes
	}
}
```

Wdyt?